### PR TITLE
Inferless Reference 

### PR DESCRIPTION
**Pull request by nickaggarwal**

### CODE REVIEW

File: references/api/inferless-input-schema.mdx
Comment (AGInfer) [inline references/api/inferless-input-schema.mdx:1]: fields

File: references/api/inferless-input-schema.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+## Input / Outputs
+
+This doc helps you configure the input / outputs of the 'infer' function in the
```
Comment (rbgo404) [inline references/api/inferless-input-schema.mdx:3]: sentence incomplete

File: references/api/inferless-input-schema.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+## Input / Outputs
+
+This doc helps you configure the input / outputs of the 'infer' function in the 
+You have to define the **input\_schema.py** in your GitHub/Gitlab repository this will help us create the Input parameters :
```
Comment (rbgo404) [inline references/api/inferless-input-schema.mdx:4]: Gitlab --> GitLab

File: references/api/inferless-python.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+### InferlessPythonModel
+
+The class called InferlessPythonModel is the entrypoint for you ML code. This class has three methods: initialize, infer, and finalize. Let's go through each method and explain their purpose, signature, and return types.
```
Comment (rbgo404) [inline references/api/inferless-python.mdx:3]: better to make the class name bold.

File: references/cli/inferless-deploy.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+## inferless deploy 
+
+
+This command will deploy the model to the inferless server. You should have run the command 'inferless init' and have the inferless.yaml before running this command.
```
Comment (rbgo404) [inline references/cli/inferless-deploy.mdx:4]: surround "inferless init" with backticks (`)

File: references/cli/inferless-init.mdx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+Use this command to initialize a new model import.
+
+### Usage
+
+```bash
+inferless init
+```
+
+You will be basked for the follwing information:
```
Comment (rbgo404) [inline references/cli/inferless-init.mdx:12]: Wrong spelling. basked --> asked

File: references/cli/inferless-model.mdx [hunk 6]
```diff
@@ -0,0 +1,62 @@
+## inferless model
+
+You can use this to manage your models.
+
+### Commands
+
+- `list`: List all models in the current workspace
```
Comment (rbgo404) [inline references/cli/inferless-model.mdx:7]: full stop missing

File: references/cli/inferless-runtime.mdx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+**Usage**:
+
+```console
+$ inferless runtime COMMAND [OPTIONS]  [ARGS]...
+```
+
+**Commands**:
+
+* `list`: list all runtimes.
+* `select [options]`: use an existing runtime with the current model
```
Comment (rbgo404) [inline references/cli/inferless-runtime.mdx:14]: full stop missing

File: references/cli/inferless-runtime.mdx [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+* `upload [options]`: create a new runtime on inferless with yaml.
+
+
+## Example
+
+**Usage**:
+
+list all the runtime in the workspace.
```
Comment (rbgo404) [inline references/cli/inferless-runtime.mdx:22]: The first letter should be capital

File: references/cli/inferless-runtime.mdx [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+```console
+$ inferless runtime list 
+```
+
+select a existing runtime for the model in the current directory.
```
Comment (rbgo404) [inline references/cli/inferless-runtime.mdx:28]: The first letter should be capital

File: references/cli/inferless-runtime.mdx [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+```console
+$ inferless runtime select -i runtime_id
+```
+
+select a a different runtime.yaml for the model in the current directory.
```
Comment (rbgo404) [inline references/cli/inferless-runtime.mdx:34]: The first letter should be capital

File: references/cli/inferless-volume.mdx [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+Manage Inferless volumes 
+
+```console
+$ inferless volume COMMAND [OPTIONS] [ARGS]...
+```
+
+**Commands**:
+
+* `create`: Create a new volume
+* `cp`: Add a file or directory to a volume.
```
Comment (rbgo404) [inline references/cli/inferless-volume.mdx:12]: add full stop in all the points

File: references/cli/inferless-workspace.mdx [hunk 12]
```diff
@@ -0,0 +1,26 @@
+### inferless workspace
+
```
Comment (rbgo404) [inline references/cli/inferless-workspace.mdx:2]: Write a single line about the inferless workspace

File: references/cli/inferless-workspace.mdx [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+```bash
+inferless workspace [COMMAND] Options
+``` 
+
+If you have multiple workspaces you can use this to
```
Comment (rbgo404) [inline references/cli/inferless-workspace.mdx:7]: seems like a incomplete sentence

File: references/cli/inferless-workspace.mdx [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+
+```bash
+inferless workspace use 
+```
+
+Prompt
+
+Select a workspace: (Use tab to autocomplete) 'Name'
+
+Switched to the 'Name' workspace.
```
Comment (rbgo404) [inline references/cli/inferless-workspace.mdx:24]: Wasn't able to understand

File: references/intro.mdx [hunk 15]
```diff
@@ -0,0 +1,15 @@
+# Reference
+
+## CLI References
+
+Here, you can add information about the Command Line Interface (CLI) references for creating deployments in Inferless.
+
+Install the Inferless CLI package
+
+'''
```
Comment (rbgo404) [inline references/intro.mdx:9]: change the single quotes to backticks: ' ---> `

File: references/intro.mdx [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+pip install inferless-cli
+'''
```
Comment (rbgo404) [inline references/intro.mdx:11]: change the single quotes to backticks: ' ---> `

**Pull request closed without merging**